### ISSUE TITLE
Reproducing epsilon-param results

### ISSUE THREAD

**Issue by Niax23** [buggyyang/CDC_compression#29]
Hi there, thank you for your awesome work!  I tried to trained a ckpt of the epsilon-param method with slightly lower bpp than the ckpt provided.

First I warmed up using beta=1e-6 and alpha=0.0, other hyper-parameters provided below, 700k steps in total.
```
n_step = 700000
scheduler_checkpoint_step = 100000
log_checkpoint_step = 5000
gradient_accumulate_every = 1
lr = 4e-5
decay = 0.9
minf = 0.5
optimizer = "adam"  # adamw or adam
n_workers = 4

load_model = False
load_step = False

pred_mode = 'noise'
loss_type = "l1"
iteration_step = 20000
sample_steps = 500
embed_dim = 64
dim_mults = (1, 2, 3, 4, 5, 6)
hyper_dim_mults = (4, 4, 4)
context_channels = 3
clip_noise = "none"
val_num_of_batch = 1
additional_note = ""
vbr = False
context_dim_mults = (1, 2, 3, 4)
sample_mode = "ddim"
var_schedule = "linear"
aux_loss_type = "lpips"
compressor = "big"

data_config = {
    "dataset_name": "vimeo",
    "data_path": "/home/Vimeo90K",
    "sequence_length": 1,
    "img_size": 256,
    "img_channel": 3,
    "add_noise": False,
    "img_hz_flip": False,
}

batch_size = 4

result_root = "results"
tensorboard_root = "logs"
```
After that, I got a ckpt with bpp about 7 with high reconstructing ability. The reconstruction results are like this:

![Image](https://github.com/user-attachments/assets/077840e3-ca81-4f31-877a-90ba1746e5b2)

Then I tried to finetuned it with beta 0.35 and alpha 0.9, hyper-parameters like this.

```
n_step = 1700000
scheduler_checkpoint_step = 100000
log_checkpoint_step = 5000
gradient_accumulate_every = 1
lr = 4e-5
decay = 0.9
minf = 0.5
optimizer = "adam"  # adamw or adam
n_workers = 4

# load
load_model = True
load_step = True

# diffusion config
pred_mode = 'noise'
loss_type = "l1"
iteration_step = 20000
sample_steps = 500
embed_dim = 64
dim_mults = (1, 2, 3, 4, 5, 6)
hyper_dim_mults = (4, 4, 4)
context_channels = 3
clip_noise = "none"
val_num_of_batch = 1
additional_note = ""
vbr = False
context_dim_mults = (1, 2, 3, 4)
sample_mode = "ddim"
var_schedule = "linear"
aux_loss_type = "lpips"
compressor = "big"

data_config = {
    "dataset_name": "vimeo",
    "data_path": "/home/Vimeo90K",
    "sequence_length": 1,
    "img_size": 256,
    "img_channel": 3,
    "add_noise": False,
    "img_hz_flip": False,
}

batch_size = 4

result_root = "results"
tensorboard_root = "logs"
```

I observed a dive of the bpp during finetuning. At the beginning of the fine-tuning, the bpp of the intermediate checkpoint rapidly dropped to 0.02.The image reconstruction quality also deteriorated consequently. 

The reconstruction result of 750k(50k steps of finetuning) looks like that, it seems that the weights of the warmup stage are deteriorated.
<img width="709" alt="Image" src="https://github.com/user-attachments/assets/5b7edc2f-c7fe-4310-a6b9-e7a9b0868386" />

This situation did not improve significantly even after 500k steps of fine-tuning, and it was also observed that the loss did not decrease noticeably.

![Image](https://github.com/user-attachments/assets/c23aa5a1-abbb-4ec4-926c-40da2eda1e43)

I will be glad if you could told me whether my training process is normal, and whether there is any error  in my training setting.

**Comment by buggyyang** [comment]
Emmm.. I have never seen this before. How about x-parameterization, it has the same issue? If you use my checkpoints (epsilon-param), I guess the result looks fine?

**Comment by Niax23** [comment]
Well, I have successfully obtained the checkpoint with the lower bpp that I desired by gradually fine-tuning the pre-trained weights you provided using slightly increased beta values. Based on my experience, I suspect that a sudden sharp increase in the beta value would severely impair the model's ability to reconstruct images from its weights. Therefore, a reasonable approach for training high compression rate models might be to progressively increase the beta value during the training process. In any case, thank you for your reply!
